awx/api/generics.py

```diff
@@ -548,6 +548,15 @@ def get_description_context(self):
         })
         return d
 
+    def get_queryset(self):
+        if hasattr(self, 'parent_key'):
+            # Prefer this filtering because ForeignKey allows us more assumptions
```

Comment:
We have a reverse ForeignKey relationship available, and that's what this would have historically used. Say you want WFJTs in an organization. `org.workflows.all()`, good. But that reverse relationship has changed, now it's all unified job templates in the organization. You can still use that `org.unifiedjobtemplates.all()`... but that's not accurate for that particular view. That would return a queryset of all UJTs in the org, not what you want for the list of workflows in the org.

I'm leveraging the fact that `WorkflowJobTemplate.objects.filter(organization=org)` works the same now as it did before. And it's arguably better in several ways. You don't introduce the `.distinct()` to the queryset, and it's just a lot cleaner.

Listing by the reverse relationship name is absolutely needed, specifically for many-to-many relationships, which is mainly what this was written for. But if you know it's a reverse FK (which it is if `parent_key` can be set), then I can't think of any reason to favor the more complex filter over this one, speaking in general terms for all views (it also happens to solve my problems here).